events/2014.json

```diff
@@ -0,0 +1,194 @@
+{
+  "name": "TechWeek",
+  "location": "Dublin City University",
+  "start": 3,
+  "end": 7,
+  "month": "March",
+  "year": 2014,
+  "live": "2014-03-03T12:00:00",
```

Comment:
Would Unix timestamp be more appropiate? I am not sure is this representation understandable by languages other than JS.